public/modules/dashboards/services/share.client.service.js

```diff
@@ -11,23 +11,19 @@ angular.module("dashboards").factory("shareService", [
      * @returns {String}
      *  */
     function createFullUrl(
-      viewType,
       country,
       admlevel,
       metric,
       parent_code,
       disaster,
-      event
+      event,
```

Comment:
Hmm. now I see this code in GitHub-syntax-highligted-colors... I think there can be an issue with this parameter/variable-name.. As "event" could be a reserved word in JS...